preloop.gms

```diff
@@ -156,9 +156,9 @@ VMExtF.FX(runCy,YTIME)$((not An(YTIME)) $(ord(YTIME) gt 1)  ) = ( iTransChar(run
 VMExtV.FX(runCy,YTIME)$(not An(YTIME)) = iDataPassCars(runCy,"PC","MEXTV");
 
 VScrRate.UP(runCy,YTIME) = 1;
-* FIXME: VScrRate.FX(runCy,YTIME), only the line of code below
-* author=giannou
-VScrRate.FX(runCy,YTIME) = iDataPassCars(runCy,"PC","scr"); 
+VScrRate.FX(runCy,YTIME) = 0.1; 
+* FIXME derive elasticities per country
```

Comment:
this doesn't seem right, the line below the comment does not contain elasticities

Reply:
FIXME comment reworked.